dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/TracingInterceptorTest.kt

```diff
@@ -224,6 +258,7 @@ internal open class TracingInterceptorTest {
         whenever(parentSpan.context()) doReturn parentSpanContext
         whenever(mockSpanBuilder.asChildOf(parentSpanContext)) doReturn mockSpanBuilder
         fakeRequest = forgeRequest(forge) { it.tag(Span::class.java, parentSpan) }
+        doAnswer { true }.whenever(mockDetector).isFirstPartyUrl(HttpUrl.get(fakeUrl))
```

Comment:
wasn't this already set up in the `set up` method ?

Reply:
Not with this value of `fakeUrl`